zipline-security/README.md

```diff
@@ -0,0 +1,3 @@
+# Zipline Security
```

Comment:
"Security" as a name is a bit nebulous to me. What are the plans for this module in the future such that it deserves to be its own module (as opposed to built-in to Zipline's core)? Maybe a future tink dependency or something?

Based on the sentence below, I would have expected "Cryptography" to be the name.

Reply:
+1 on changing to cryptography.

Reply:
Yea, I agree with this. We started with cryptography but changed to security on a whim. Changing